### DISCUSSION TITLE
Question: How to place hold on a card payment and capture less payment later?

### DISCUSSION THREAD

**Discussion by EngrSalim** [flutter-stripe/flutter_stripe#1741] [Q&A]
Hello i am using Card Payment, options using the authorize and capture later option documented here : https://stripe.com/docs/payments/capture-later
we have authorize payment of 50$ and want to capture less amount then the authorize amount but stripe sdk for flutter does not allow me to do this

**Comment by MrTONYCHAN** [comment]
This operation should be performed by your backend, not directly by the Flutter client.

Create the PaymentIntent on the server with manual capture:

```js
const intent = await stripe.paymentIntents.create({
  amount: 5000,
  currency: "usd",
  capture_method: "manual",
});
```

Use the returned client secret with `flutter_stripe` to confirm the payment. After authorization, the PaymentIntent should reach `requires_capture`.

To capture a smaller amount, call Stripe from your backend:

```js
await stripe.paymentIntents.capture(paymentIntentId, {
  amount_to_capture: 3500,
});
```

Here `5000` authorizes $50 and `3500` captures $35. The remaining authorized amount is released; it is not available for a later second capture under the normal single-capture flow.

Do not place the Stripe secret key in the Flutter application. Send the PaymentIntent ID and desired final amount to an authenticated backend endpoint, validate that the user/order owns that PaymentIntent, ensure the amount does not exceed the capturable amount, and then perform the capture server-side.

Also capture before the authorization expires. The exact authorization window depends on the payment method and card network. Listen for PaymentIntent webhooks and make the capture endpoint idempotent so retries cannot create inconsistent order state.
